xml/installation-kvm_xpointer.xml

```diff
@@ -223,7 +223,7 @@
 <screen>export LC_ALL=C</screen>
     <note>
      <para>
-      This can be added to <literal>~stack/.bashrc</literal> or
+      This can be added to <literal>~/.bashrc</literal> or
```

Comment:
Is this change based upon a bug or is the user always logged in as "stack" or what has changed here?

Reply:
bsc#1082644 - [doc] Notes on overall document [FEB 28]
which refers to
https://etherpad.nue.suse.com/p/cloud8-installation-guide-notes
@125 10.3.1 Installing cloud lifecycle manager
- links and directions on licencing are incorrect
command path need to be corrected.
This can be added to ~stack/.bashrc -> This can be added to ~/.bashrc 